types/module/module.go

```diff
@@ -52,8 +52,8 @@ type AppModuleBasic interface {
 
 	// client functionality
 	RegisterRESTRoutes(client.Context, *mux.Router)
-	GetTxCmd(client.Context) *cobra.Command
-	GetQueryCmd(*codec.Codec) *cobra.Command
+	GetTxCmd(clientCtx client.Context) *cobra.Command
+	GetQueryCmd(clientCtx client.Context) *cobra.Command
```

Comment:
```suggestion
	GetQueryCmd(client.Context) *cobra.Command
```

Reply:
Adding the variable name is actually helpful when implementing the method using code completion because we wanted clientCtx rather than ctx. But I can remove it if you prefer

Reply:
Where would one get `ctx` from? I'd opt to remove it personally. Not blocking though, so proceed at your own discretion.

Reply:
Sorry it would be `context`. I'd prefer to keep it because it helped my IDE (goland) keep the style conventions from #6290.